<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IOWidget</class>
 <widget class="QWidget" name="IOWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>916</width>
    <height>826</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>IOWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_4">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>IO Status</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>890</width>
            <height>781</height>
           </rect>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0">
           <property name="sizeConstraint">
            <enum>QLayout::SetDefaultConstraint</enum>
           </property>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="5,10">
             <property name="spacing">
              <number>15</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_in" native="true">
               <property name="minimumSize">
                <size>
                 <width>408</width>
                 <height>546</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_out" native="true">
               <property name="minimumSize">
                <size>
                 <width>403</width>
                 <height>546</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">background-color: rgb(255, 255, 255);</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Setting</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>408</x>
         <y>490</y>
         <width>223</width>
         <height>37</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pushButton_save">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Save</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_cancel">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>408</x>
         <y>30</y>
         <width>233</width>
         <height>21</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="radioButton_in">
          <property name="text">
           <string>Input</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radioButton_out">
          <property name="text">
           <string>Output</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QListWidget" name="listWidget_IO">
       <property name="geometry">
        <rect>
         <x>96</x>
         <y>22</y>
         <width>237</width>
         <height>511</height>
        </rect>
       </property>
       <property name="contextMenuPolicy">
        <enum>Qt::CustomContextMenu</enum>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>508</x>
         <y>70</y>
         <width>135</width>
         <height>375</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="lineEdit_name"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_cardtype"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_cardindex"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_card_node"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_IOIndex"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_sense"/>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_group"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>407</x>
         <y>71</y>
         <width>86</width>
         <height>373</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="spacing">
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>IO Name</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Card Type</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Card No</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Node No（Can）</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>IO Index</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Sense</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Group</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_save</sender>
   <signal>clicked()</signal>
   <receiver>IOWidget</receiver>
   <slot>onBtnSaveClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>462</x>
     <y>544</y>
    </hint>
    <hint type="destinationlabel">
     <x>442</x>
     <y>594</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_cancel</sender>
   <signal>clicked()</signal>
   <receiver>IOWidget</receiver>
   <slot>onBtnCancelClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>591</x>
     <y>539</y>
    </hint>
    <hint type="destinationlabel">
     <x>716</x>
     <y>589</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onBtnCancelClicked()</slot>
  <slot>onBtnSaveClicked()</slot>
 </slots>
</ui>
